### PR TITLE
Implement Importer For Immuniweb scan parser

### PR DESCRIPTION
**Pull request by propersam**
Issue #1185 

Please submit your pull requests to the 'dev' branch.

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant (DefectDojo's code isn't currently flake8 compliant, but we're trying to correct that.)
- [x] Model changes should include the necessary migrations in the dojo/dd_migrations folder.
- [x] Add applicable tests to the unit tests.

### CODE REVIEW

**Comment by propersam** [comment]
someone with Admin privilege pushed code directly to the Dev Branch without properly implementing Flake8 .

File: dojo/forms.py [hunk 1]
```diff
@@ -484,7 +486,7 @@ class ScanSettingsForm(forms.ModelForm):
         help_text=addHelpTxt,
         validators=[
             validators.RegexValidator(
-                regex='^\s*([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+,*\s*)+\s*$',
+                regex=r'^\s*([0-9]+\.[0-9]+\.[0-9]+\.[0-9]+,*\s*)+\s*$',
```
Comment (devGregA) [inline dojo/forms.py]: @propersam was this change intentional?

Comment (propersam) [reply dojo/forms.py]: Yes, It was flagging flake8 error at the '\'. adding 'r' solves it.
